tests/test_debugging.py

```diff
@@ -273,48 +273,6 @@ def circuit():
 
         _compare_numpy_dicts(result, expected)
 
-    @pytest.mark.parametrize("diff_method", [None, "backprop", "parameter-shift", "adjoint"])
```

Comment:
I'd say we should probably keep this test around and only remove it when we remove `default.qubit.legacy`.